examples/airflow/docker_compose_decorator.py

```diff
@@ -20,6 +20,9 @@
     ]
 )
 
+# Dont load Airflow example DAGs because they cause visual pollution
+docker_compose["x-airflow-common"]["environment"]["AIRFLOW__CORE__LOAD_EXAMPLES"] = "false"
```

Comment:
Nice 👍  Let's move this into `Dockerfile.template` : https://github.com/TobikoData/sqlmesh/blob/main/examples/airflow/Dockerfile.template#L49

Reply:
Do you mean add an ENV entry to `Dockerfile.template` to set it?

The problem is, it gets overridden again the second the containers are instantiated, because its explicitly defined in the upstream `docker-compose.yml` file